packages/manager/modules/pci/src/projects/project/storages/databases/components/tags-input/tags-input.html

```diff
@@ -1,11 +1,12 @@
 <form ng-submit="$ctrl.addTag()">
-    <oui-field data-label="{{ $ctrl.label }}" data-size="xl">
-        <div class="oui-input-group oui-input-group_xl">
+    <oui-field data-label="{{ $ctrl.label }}">
+        <div class="oui-input-group">
             <input
                 class="oui-input"
                 type="text"
-                id="tagInput"
-                name="tagInput"
+                id="{{'tagInput-' + $ctrl.id}}"
+                name="{{'tagInput-' + $ctrl.id}}"
```

Comment:
I noticed that you used lodash `uniqueId`.
May personal approach to deals that and to avoid to use external library is:

```suggestion
                data-ng-init="tagInputId = 'whateverYouWant'"
                data-ng-attr-id="{{tagInputId}}"
				data-ng-attr-name="{{tagInputId}}"
```

I hope it could be help you on future dev.